torch/_dynamo/variables/nn_module.py

```diff
@@ -289,7 +288,7 @@ def call_function(
             # If we are tracing the higher order op, we want Dynamo to step
             # inside the module call so that Dynamo can see the underlying
             # parameters and buffers and raise them as inputs to the graph.
-            if tx.output.is_root_tracer() and is_allowed(mod.__class__):
+            if tx.output.is_root_tracer() and mod.__module__.startswith("torch.nn."):
```

Comment:
We are going to filter out these torch builtin modules and put a ```call_module``` node in the graph, so I think this is equivalent with the former way. 
The annotation is for higher order ops tracing, where ```tx.output.is_root_tracer()``` returns False and we step inside the module.